### PR TITLE
Added code to display Total Portfolio Value

### PR DESCRIPTION
**Pull request by Sashank999**
Just added the code required to display the total stock value like this :

Stock Exchange _ TORN - Google Chrome 28-Dec-20 10_06_24 AM

### CODE REVIEW

File: torntools/scripts/content/stockexchange/ttStockexchange.js [hunk 1]
```diff
@@ -481,3 +481,21 @@ function getDiffClass(diff) {
...
+let totalPortfolioValue = 0;
+// Loops through all the li elements (stock blocks)
+for (let element of document.findAll('.stock-main-wrap>ul.stock-cont>li.item-wrap')) {
+	totalPortfolioValue += parseInt(element.find('.item .info .c-price-wrap .first-row span').innerText.replace('Worth: ', '').replace('$', '').replace(/,/g, '').trim());
+}
+totalPortfolioValue = new Intl.NumberFormat('en-US', {
+	style: 'currency',
+	currency: 'USD',}).format(totalPortfolioValue).slice(0, -3);
+document.querySelector("#info_wrapper").insertAdjacentElement("beforeBegin", totalPortfolio);
+totalPortfolio.style.fontSize = "18px;
```
Comment (DeKleineKobini) [inline torntools/scripts/content/stockexchange/ttStockexchange.js:499]: Honestly, I don't like it being bigger than literally anything else in TT. I would just remove this.

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]
The entire location of this code is wrong.
It's in the right file, but you can't just put it at the bottom and assume it will work.

**Comment by Mephiles** [comment]
Also, could you move the value into the black "Your portfolio" heading as this will just remove clutter and unnecessary containers. It could look like this: 'Your portfolio ($1,000,000)'

**Comment by Sashank999** [comment]
Now I have changed the code. Thanks to Mephiles for the suggestions in removing some code. Now it looks like this :
Stock Exchange _ TORN - Google Chrome 29-Dec-20 8_33_15 PM

**Comment by Sashank999** [comment]
Added changes as per the TT Standards.

**Pull request merged**